src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -175,10 +176,11 @@ private void createDatePanel() {
                                            I18nUtils.getLocale()
          									+
          									" for Today: ");
-        datePanel_.setBorder(tb);
-        datePanel_.setToolTipText("Displays today's date and time.");
+        datePanel2_.setBorder(tb);
+        datePanel2_.setToolTipText("Displays today's date and time.");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) active Embedded Strings : Method: setToolTipText "Displays today's date and time." [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=gyzr%3AGlobalyzerES1Check)